hal/motor_hal.hpp

```diff
@@ -16,10 +16,14 @@ enum MotorStatus {
 class Motor {
 private:
     pthread_mutex_t lock_motor;
+    Coord r;
```

Comment:
Would it be possible to add comments specifying what `r` and `theta` are?  If a description using words is too unwieldy, would it be possible to link to a figure?

Reply:
+1, use doxygen compliant comments on fields.